app/views/content/life-as-a-teacher/change-careers/benefits-of-changing-careers.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Benefits of changing careers
+description: |-
+  Benefits of changing careers
+layout: "layouts/minimal"
+colour: pastel yellow-yellow
+keywords:
+  - teaching
+  - career changer
+  - changing careers
+  - career
+content: 
+  - "content/life-as-a-teacher/change-careers/benefits-of-changing-careers/header" 
+  - "content/life-as-a-teacher/change-careers/benefits-of-changing-careers/article"
+  - "content/life-as-a-teacher/how-to-become-a-teacher-cta"
```

Comment:
Is this the right CTA here? We have a different how page for career changers?

Should we have advisers instead?